features/attached_enable.feature

```diff
@@ -945,7 +945,6 @@ Feature: Enable command behaviour when attached to an Ubuntu Pro subscription
 
     @series.xenial
     @series.bionic
-    @uses.config.contract_token
```

Comment:
Should we remove that line on this test ?

Reply:
This line is useless here, in the `Scenario`, because the `Feature` has the decorator. (And here it makes sense , because "attached_enable" implies that all scenarios will always need the token)